src/desktop/apps/auction_support/templates/conditions-of-sale-checkbox.jade

```diff
@@ -0,0 +1,9 @@
+.registration-form-section.accept-cos
```

Comment:
#minor but I believe we typically use underscores for filenames, so this file should be `conditions_of_sale_checkbox`